index.js

```diff
@@ -88,11 +88,14 @@ function fastifyWebsocket (fastify, opts, next) {
   // to monkeypatching for now.
   const oldClose = fastify.server.close
   fastify.server.close = function (cb) {
+    // Call oldClose first so that we stop listening. This ensures the
+    // server.clients list will be up to date when we start closing below.
+    oldClose.call(this, cb)
```

Comment:
The effect of this is that node stops accepting new connections before we start closing any, which means there's no window where new connections can come in after we think we've closed them all, right? https://nodejs.org/api/net.html#net_server_close_callback